ecl2df/summary.py

```diff
@@ -422,6 +424,30 @@ def df(
     return dframe
 
 
+def _df2pyarrow(dframe) -> pyarrow.Table:
+    """Construct a Pyarrow table from a dataframe, conserving metadata"""
+
+    # Datatypes to use for DATE column and all the numeric columns
+    # dt_timestamp_ms = pyarrow.timestamp("ms")
+    dt_timestamp_ms = pyarrow.timestamp("ns")  # fixme!
```

Comment:
yes, fixme. Probably need to detect the timestamp-type in the dataframe or always convert to milliseconds if we so decide.